### ISSUE TITLE
openSUSE Tumbleweed with GNOME 44: google-chrome settings button not working

### ISSUE THREAD

**Issue by rafaeluy** [ubuntu/gnome-shell-extension-appindicator#422]
With only this extension installed, the google-chrome settings menu (top right) doesn't work. However, if the dash to panel extension is added, in this case it does work.

**Comment by 3v1n0** [comment]
It's not totally clear to me, could you provide some screenshot?

**Comment by rafaeluy** [comment]
More information about the extension issue in openUSE Tumbleweed.

Normally, openSUSE users use the packman repository to obtain non-existing proprietary drivers for the distribution. packman is not an official openSUSE repository, but is still recommended for use by openSUSE.

Problems I have encountered with several installations of openSUSE Tumbeweed Gnome 44:

Installing it on an AMD system (I don't use Intel in my computers, as I don't use any NVidia video card) with the default installation, everything works correctly, including the extension.

When adding the proprietary drivers, MESA is replaced by the existing packman version (which has among other things the acceleration support removed in the openSUSE packaged version, as does Fedora). In this case, when you try to use google-chrome or microsoft edge, with only the gnome-shell-extension-appindicator extension installed, you get the error of not being able to activate the configuration menu (the google-chrome configuration menu is located at the top right, and are the three dots there, where clicking on it does not get any result. The same happens when you try to access with the mouse to the configuration menu located in Edge at the top right (the three dots). This happens when the browser screen is maximized.

The problem is corrected by installing the dash to panel extension.

If I install in VirtualBox, I don't get any problem, neither with the default installation, nor with the proprietary packman drivers.
Which leads me to think that the problem could be with AMD's proprietary drivers.

To sum up: the extension you have developed (a must have extension for any Gnome user) has no problem, the problem is in the existing Mesa compilation in the packman repository (at least that's what it seems), because it is when you replace the existing default ones in openSUSE that the problem occurs.

I am sorry for not responding sooner, because I have been testing various installations and their results.

And I am sorry for posting such an information poor error.

Sorry for my English, but my native language is Spanish.

**Comment by 3v1n0** [comment]
What's not clear to me is "what does not work?"

Can you provide screenshots, logs and furhter infos to make it clearer in what sense the extension is not working?
Thanks!

**Comment by 3v1n0** [comment]
Ah.. Si quieres, puedes escribir en español... No hay problemas!

**Comment by rafaeluy** [comment]
Pude identificar el problema: es AnyDesk. Yo tenía instalado el serviciod e AnyDesk, que deja ewl ícono en el system tray, y eso produce el error al intentar abrie el menú de los navegadores situado arriba a la derecha (y cuando los navegadores se encuentran a pantalla completa, yq que de lo contrario, todo funciona correctamente).

Al desactivar el servicio de AnyDesk y reiniciar el sistema, todo funciona normalmente.

**Comment by okarika** [comment]
The problem is when the extension displays the icon of anydesk. It somehow overlaps the screen, like you can see it in the video: (sorry gnome cant capture the mouse cursor in a video apparently) reproduced on gnome 44
[Screencast from 2023-04-14 12-36-47.webm](https://user-images.githubusercontent.com/21237222/232023366-c2eb87fe-9dfa-4bb2-a49d-8fa416842708.webm)
by removing anydesk, the problem cannot be reproduced
